src/sqlancer/arangodb/ArangoDBQueryAdapter.java

```diff
@@ -0,0 +1,15 @@
+package sqlancer.arangodb;
+
+import sqlancer.common.query.Query;
+
+public abstract class ArangoDBQueryAdapter extends Query<ArangoDBConnection> {
+    @Override
+    public String getQueryString() {
+        throw new UnsupportedOperationException();
```

Comment:
This is actually just to make sure that this function is not called, because it is a SQL function. Now with ArangoDB, it makes again sense to support it, just logging is done differently and not through this function, so I avoided logging twice like this.

I added a comment that this function should not be used, but I can switch it to implementing it, if you want.